=== RUN   TestH2c
[GIN] 2025/06/20 - 00:11:13 | 200 |       9.882µs |       127.0.0.1 | GET      "/"
    gin_test.go:98: accept tcp 127.0.0.1:45363: use of closed network connection
--- PASS: TestH2c (0.00s)
PASS
ok  	github.com/gin-gonic/gin	0.007s
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/gin_test.go
TestName:  TestH2c
Import added at line: 7
Header added at line: 85
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/gin_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestH2c
[GIN] 2025/06/20 - 00:11:13 | 200 |      64.764µs |       127.0.0.1 | GET      "/"
    gin_test.go:103: accept tcp 127.0.0.1:34929: use of closed network connection
--- PASS: TestH2c (0.11s)
PASS
ok  	github.com/gin-gonic/gin	0.116s
Bugreport info: 0_1,suc
Bugreport info: 0_2,fail
Bugreport info: 0_3,fail
Bugreport info: 0_4,fail
Bugreport info: 0_5,fail
Bugreport info: 0_6,fail
Bugreport info: 0_7,fail
Bugreport info: 0_8,fail
Bugreport info: 0_9,fail
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/gin_test.go
TestName:  TestH2c
Import added at line: 7
Header added at line: 85
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/gin_test.go
=== RUN   TestH2c
Reading trace from rewrittenTrace_1

Exit Replay with code  10 Timeout
Exit Message: panic: Timeout

FAIL	github.com/gin-gonic/gin	60.068s
FAIL
